lib/pure/unicode.nim

```diff
@@ -207,28 +226,39 @@ template fastToUTF8Copy*(c: Rune, s: var string, pos: int, doInc = true) =
     discard # error, exception?
 
 proc toUTF8*(c: Rune): string {.rtl, extern: "nuc$1".} =
-  ## Converts a rune into its UTF-8 representation
+  ## Converts a rune into its UTF-8 representation.
+  runnableExamples:
+    let a = "añyóng"
+    doAssert a.runeAt(1).toUTF8 == "ñ"
+
   result = ""
   fastToUTF8Copy(c, result, 0, false)
 
 proc add*(s: var string; c: Rune) =
+  ## Adds a rune ``c`` to a string ``s``.
+  runnableExamples:
+    var s = "abc"
+    let c = "ä".runeAt(0)
+    s.add(c)
+    doAssert s == "abcä"
+
   let pos = s.len
   fastToUTF8Copy(c, s, pos, false)
 
 proc `$`*(rune: Rune): string =
-  ## Converts a Rune to a string
+  ## Converts a Rune to a string.
```

Comment:
I think it's confusing that the docs for `$` and `toUTF8` says different things when they are just aliases.

Reply:
Which version do you prefer: "Converts a Rune to a string." or "Converts a rune into its UTF-8 representation."? Or something else?

I'll change documentation of `toUTF8` to have the preferred version. For `$` I'll put "Alias for `toUTF8`".

Reply:
I prefer "Converts a rune into its UTF-8 representation." since it's more consistent with the rest of the unicode docs, and "Converts a Rune to a string" doesn't really say anything that isn't obvious from the proc signature anyway.